src/vcbuild/dmd.visualdproj

```diff
@@ -543,6 +543,7 @@
    <File path="..\dmd\scanmscoff.d" />
    <File path="..\dmd\scanomf.d" />
    <File path="..\dmd\semantic.d" />
+   <File path="..\dmd\semantic2.d" />
    <File path="..\dmd\sideeffect.d" />
```

Comment:
Did we forget to add `..\dmd\semantic3.d` in #7544?

Reply:
Yes. https://github.com/dlang/dmd/pull/7547